deployment/terraform/engine.go

```diff
@@ -40,6 +40,38 @@ func (t *Terraform) runCommand(dst io.Writer, args ...string) error {
 	mlog.Debug("Running terraform command", mlog.String("args", fmt.Sprintf("%v", args)))
 	cmd := exec.CommandContext(ctx, terraformBin, args...)
 
+	return _runCommand(cmd, dst)
+}
+
+func (t *Terraform) attachSG() error {
+	awsBin := "aws"
```

Comment:
Not opposed to this for time's sake but should we consider the more programmatic way of integrating with the official Golang AWS SDK as a possible future improvement?

Reply:
Yep, you already have https://github.com/mattermost/mattermost/issues/22150. We can add this one to that as well.